pagopa/io-app
# chore: [IOBP-561] Addition of new payments page playground

## Short description
added playground entry in profile screen

## List of changes proposed in this pull request
- added navigator entries
- added profile playground entry

## How to test
with debug/DEV mode on, navigate to the profile screen, then to playgrounds and make sure that clicking "payments page playground" opens a blank screen that allows back navigation
